src/labels/CustomLabels.labels

```diff
@@ -2897,6 +2919,20 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Message warning user if totals don&apos;t match expected</shortDescription>
         <value>Require Expected Totals Match is selected but the Expected Count of Gifts and Expected Total Batch Amount fields are blank. Edit the batch details and either uncheck Require Expected Totals Match or add values in the related fields.</value>
     </labels>
+    <labels>
+        <fullName>bgeBatchDryRun</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Text for batch dry run in BGE.</shortDescription>
+        <value>Batch Dry Run</value>
+    </labels>
+    <labels>
+        <fullName>bgeBatchDryRunProcessing</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Text for toast when batch dry run is started</shortDescription>
+        <value>Please don't navigate out of this page.</value>
```

Comment:
Recommend to revise to:

Batch Dry Run in progress. Please stay on this page until it's done processing.